guides/common/modules/snip_warning-destroy-vm-on-host-delete.adoc

```diff
@@ -0,0 +1,8 @@
+[WARNING]
+====
+By default, if a host entry that is associated with a virtual machine is deleted, the virtual machine will remain on the compute resource.
+
+If you want {Project} to destroy virtual machines on the compute resource when the associated host entry is deleted, navigate to *Administer* > *Settings* > *Provisioning* and set the *Destroy associated VM on host delete* setting to *Yes*.
+
+To avoid destroying virtual machines in this situation, disassociate the virtual machine from {Project} without removing it from the compute resource or keep the setting set to *No*.
```

Comment:
I don't think you need this sentence because you have already explained the default behavior.